src/main/java/duke/main/Storage.java

```diff
@@ -0,0 +1,30 @@
+package duke.main;
+import duke.exception.*;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.io.File;
+
+public class Storage {
+    private TaskList taskList;
+    private String path;
+
+    public Storage(TaskList taskList, String path) throws FileNotFoundException {
+        this.taskList = taskList;
+        this.path = path;
```

Comment:
hey, what do you think about renaming path to something like filePath? It'll make it crystal clear what it's all about.